source/features/keyboard-navigation.tsx

```diff
@@ -0,0 +1,67 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {isEditable} from '../helpers/dom-utils';
+
+const shortcutClass = new Map<string, string>([
+	['e', '.unminimized-comment [aria-label="Edit comment"]:not([hidden])'],
+	['d', '[aria-label="Delete comment"]']
+]);
+
+function runShortcuts(event: KeyboardEvent): void {
+	if (isEditable(event.target)) {
+		return;
+	}
+
+	const focusedComment = select(':target')!;
+
+	if (['j', 'k'].includes(event.key)) {
+		event.preventDefault();
+
+		const items = select.all('.js-targetable-element')
+			.filter(comment => !comment.querySelector('.minimized-comment:not(.d-none)'));
```

Comment:
This filter does not work correctly it needs to check also for `'.js-minimizable-comment-group'`.
Can I do the following?
 
```suggestion
		const itemsSelectors = pageDetect.isPRFiles ? '.js-targetable-element' : '.js-minimizable-comment-group'
		const items = select.all(itemsSelectors)
			.filter(comment => !comment.querySelector('.minimized-comment:not(.d-none)'));
```

Reply:
If by **"does not work correctly"** you mean **"it highlights reviews but they don't have a highlight outline"**, then the solution could be:

`.js-targetable-element:not([id^="pullrequestreview"])`

Reply:
No mean that minimized comments are highlighted

Reply:
That's fine. Minimized comments are still comments

Reply:
> `.js-targetable-element:not([id^="pullrequestreview"])`

Let's also use this